spec/features/create_media_video_spec.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+require 'random_word'
+
+RSpec.describe 'Create Media Video object', type: :feature do
+  include ActiveJob::TestHelper # have background jobs run synchronously
```

Comment:
The comment here is misleading, this provides `perform_enqueued_jobs`, the synchronousness is defined in `config/environments/test.rb`

Reply:
gotcha - thanks.